VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -158,6 +159,21 @@ else if (!Config.getInstance().getGoogleConfig().isEnabled()) {
         if(customTitle!=null){
             customTitle.setText(getString(R.string.login_title));
         }
+
+        TextView version_tv = (TextView)  findViewById(R.id.tv_version_no);
+        try{
+            String envDisplayName = Config.getInstance().getEnvironmentDisplayName();
+
+            if(envDisplayName != null && envDisplayName.length() > 0 ) {
+                version_tv.setVisibility(View.VISIBLE);
+                String versionName = PropertyUtil.getManifestVersionName(this);
+                String text = String.format("%s %s %s",
+                        getString(R.string.label_version), versionName, envDisplayName);
+                version_tv.setText(text);
+            }
+        }catch(Exception e) {
+            logger.error(e);
+        }
```

Comment:
@hanningni Same code is already there in the <code>NavigationFragment</code>. We either factor this duplication or are we not going to show the version in <code>NavigationFragment</code> ?